### ISSUE TITLE
NullReferenceException on access to a cell text

### ISSUE THREAD

**Issue by volikag** [JanKallman/EPPlus#172]
After updating to v4.5.1 from v4.1.1 I've got NullReferenceException for some files (when other files parses as expected).
Problem report was generated with SQL Server Reporting Services and contains just data and formatting (no formulas).
When I downgrade back to 4.1.1 this report parses normally.

Report contains personal data, so I can't attach it.
And when I remove personal data (and save it with Excel), exception stops throwing for overwritten file.

Exception thrown when I try to access to property **cell.Text** from this code (in if statement):
```
using (ExcelPackage xlPackage = new ExcelPackage(fileInfo))
            {
                var worksheet = xlPackage.Workbook.Worksheets.First();

                foreach (var cell in worksheet.Cells)
                {
                    if (cell != null && cell.Text != string.Empty)
                    {
                        ConsoleWriteLine(cell.Text);
                    }
                }
```

Result StackTrace:
```
System.NullReferenceException: Object reference not set to an instance of an object.
   at OfficeOpenXml.ExcelRangeBase.GetFormattedText(Boolean forWidthCalc)
   at OfficeOpenXml.ExcelRangeBase.get_Text()
   at ReportHelper.GetCells(FileInfo fileInfo) in ...
```

**Comment by JanKallman** [comment]
Interesting, but if I can't reproduce the issue, I really can't say what's wrong. Try to isolate the execption to a cell to see what data it consists of. And then attach a reproducible sample from that here.

**Comment by volikag** [comment]
I've made further investigation and find out that exception in GetFormattedText thrown due to 
'ExcelNumberFormatXml.ExcelFormatTranslator nf' remains to be null, because nfID in CellXfs is set to -2147483648 and no FormatTranslator selected.

CellXfs for selected style contains
`-2147483648|-2147483648|2|0|0Top|General|True|LeftToRight|False0TrueFalseFalse0False`

**Comment by linchenrr** [comment]
I have got the same Problem, too.  
Here is the file can cause the Exception.

[quest.xlsx](https://github.com/JanKallman/EPPlus/files/1854425/quest.xlsx)

**Comment by JanKallman** [comment]
This occurs the numberformat is invalid or not found. I'll add a fix in the next commit.

**Comment by saad749** [comment]
Any update on this?

**Comment by kchristman54** [comment]
If you need a file to test against, I have some files I can share directly with you...Let me know.  (While these files don't contain sensitive information, I'd rather not publicize them directly on the web.)

We are having the same problem with v.4.5.1, and are downgrading to 4.1.1 until fixed.

**Comment by linchenrr** [comment]
It's already fixed in the master branch.

**Comment by saad749** [comment]
Will the nuget package be updated soon?

**Comment by JanKallman** [comment]
Fixed in 4.5.2

**Comment by wagramos** [comment]
Is it fixed for .net core as well?
